.cloudbuild/seqr-docker.cloudbuild.yaml

```diff
@@ -0,0 +1,12 @@
+steps:
```

Comment:
It looks like this no longer has an automated trigger hooked up to it - is that correct?

Reply:
agreed, with the way we currently handle static assets we can't have an automated build from the dev branch